packages/react/src/stories/ic-toast.stories.js

```diff
@@ -417,10 +417,10 @@ export const Playground = {
     },
 
     variant: {
-      options: ["neutral", "info", "warning", "error", "success", ""],
+      options: ["neutral", "info", "warning", "error", "success", "ai", ""],
 
       control: {
-        type: "select",
+        type: "inline-radio",
```

Comment:
Very tiny thing - is it worth making this update in the web-components playground as well to match?

Same with the alert